src/org/zaproxy/zap/utils/FontUtils.java

```diff
@@ -121,6 +129,27 @@ public static Font getFont (String name) {
 	public static Font getFont (int style) {
 		return getDefaultFont().deriveFont(style);
 	}
+
+    /**
+     * Gets the default work panels font
+     * @return work panels font
+     */
+    public static Font getWorkPanelsFont () {
```

Comment:
btw, this could be replaced with a `getFront(FontType fontType)`, if we happen to add more types we don't need to add more methods (same for `isDefaultFontSet` and `WithFallback`).